lte/cloud/go/services/subscriberdb/digest_test.go

```diff
@@ -384,50 +384,50 @@ func TestGetPerSubscriberDigestsDiff(t *testing.T) {
 	})
 	t.Run("all empty", func(t *testing.T) {
 		prev := []*lte_protos.SubscriberDigestWithID{
-			{Sid: &lte_protos.SubscriberID{Id: "00000"}, Digest: &lte_protos.Digest{Md5Base64Digest: "apple"}},
-			{Sid: &lte_protos.SubscriberID{Id: "00001"}, Digest: &lte_protos.Digest{Md5Base64Digest: "banana"}},
-			{Sid: &lte_protos.SubscriberID{Id: "00002"}, Digest: &lte_protos.Digest{Md5Base64Digest: "cherry"}},
+			{Sid: &lte_protos.SubscriberID{Id: "00000", Type: lte_protos.SubscriberID_IMSI}, Digest: &lte_protos.Digest{Md5Base64Digest: "apple"}},
+			{Sid: &lte_protos.SubscriberID{Id: "00001", Type: lte_protos.SubscriberID_IMSI}, Digest: &lte_protos.Digest{Md5Base64Digest: "banana"}},
+			{Sid: &lte_protos.SubscriberID{Id: "00002", Type: lte_protos.SubscriberID_IMSI}, Digest: &lte_protos.Digest{Md5Base64Digest: "cherry"}},
 		}
 		next := []*lte_protos.SubscriberDigestWithID{}
 		toRenew, deleted := subscriberdb.GetPerSubscriberDigestsDiff(prev, next)
 
 		assert.Empty(t, toRenew)
-		assert.Equal(t, []string{"00000", "00001", "00002"}, deleted)
+		assert.Equal(t, []string{"IMSI00000", "IMSI00001", "IMSI00002"}, deleted)
 	})
 	t.Run("tracked empty", func(t *testing.T) {
 		prev := []*lte_protos.SubscriberDigestWithID{}
 		next := []*lte_protos.SubscriberDigestWithID{
-			{Sid: &lte_protos.SubscriberID{Id: "00000"}, Digest: &lte_protos.Digest{Md5Base64Digest: "apple"}},
-			{Sid: &lte_protos.SubscriberID{Id: "00001"}, Digest: &lte_protos.Digest{Md5Base64Digest: "banana"}},
-			{Sid: &lte_protos.SubscriberID{Id: "00002"}, Digest: &lte_protos.Digest{Md5Base64Digest: "cherry"}},
+			{Sid: &lte_protos.SubscriberID{Id: "00000", Type: lte_protos.SubscriberID_IMSI}, Digest: &lte_protos.Digest{Md5Base64Digest: "apple"}},
+			{Sid: &lte_protos.SubscriberID{Id: "00001", Type: lte_protos.SubscriberID_IMSI}, Digest: &lte_protos.Digest{Md5Base64Digest: "banana"}},
+			{Sid: &lte_protos.SubscriberID{Id: "00002", Type: lte_protos.SubscriberID_IMSI}, Digest: &lte_protos.Digest{Md5Base64Digest: "cherry"}},
 		}
 		expectedToRenew := map[string]string{
-			"00000": "apple",
-			"00001": "banana",
-			"00002": "cherry",
+			"IMSI00000": "apple",
+			"IMSI00001": "banana",
+			"IMSI00002": "cherry",
 		}
 		toRenew, deleted := subscriberdb.GetPerSubscriberDigestsDiff(prev, next)
 		assert.Equal(t, expectedToRenew, toRenew)
 		assert.Empty(t, deleted)
 	})
 	t.Run("both not empty, basic", func(t *testing.T) {
 		prev := []*lte_protos.SubscriberDigestWithID{
-			{Sid: &lte_protos.SubscriberID{Id: "00001"}, Digest: &lte_protos.Digest{Md5Base64Digest: "apple"}},
-			{Sid: &lte_protos.SubscriberID{Id: "00002"}, Digest: &lte_protos.Digest{Md5Base64Digest: "banana"}},
-			{Sid: &lte_protos.SubscriberID{Id: "00004"}, Digest: &lte_protos.Digest{Md5Base64Digest: "dragonfruit"}},
+			{Sid: &lte_protos.SubscriberID{Id: "00001", Type: lte_protos.SubscriberID_IMSI}, Digest: &lte_protos.Digest{Md5Base64Digest: "apple"}},
```

Comment:
Yeah technically IMSIs need to fit this regex (e.g. `IMSI0000000000042`), but it's not a huge deal for these types of tests https://github.com/magma/magma/blob/ea55e13ba409a868a7fa27220d3a6ae89238d2ca/orc8r/cloud/go/obsidian/swagger/v1/swagger.yml#L11425-L11429

Reply:
Yeah this is kinda a nit, just wanted to change these to 5 digits for consistency (since all other mock IMSIs in this file are 5 digits) @hcgatewood